pkgs/development/interpreters/python/cpython/2.7/default.nix

```diff
@@ -121,7 +122,7 @@ let
 
   buildInputs =
     optional (stdenv ? cc && stdenv.cc.libc != null) stdenv.cc.libc ++
-    [ bzip2 openssl zlib ]
+    [ bzip2 openssl zlib makeWrapper]
     ++ optionals stdenv.isCygwin [ expat libffi ]
     ++ [ db gdbm ncurses sqlite readline ]
     ++ optionals x11Support [ tcl tk xlibsWrapper libX11 ]
```

Comment:
Not sure to understand well : Do you want me to put makeWrapper in `[ tcl tk libX11 xproto tix ]` ?